src/unity/ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 2018.2.5f1
```

Comment:
This is a funny one - i googled this the other day and apparently it is appropriate to check it in, but its weird if this gets wound backwards I think because one of us is on a slightly different version! I think i'd vote for treating it like a high water mark, otherwise we will end up playing ProjectVersion tennis :). What do you think?